### DISCUSSION TITLE
How to access GoTrue endpoints?

### DISCUSSION THREAD

**Discussion by andrewcoelho** [supabase/supabase#3119] [Questions] [answered]
What is the full URL I use to access the GoTrue endpoints associated with my Supabase project? (https://supabase.io/docs/gotrue/server/about#endpoints)

Using my Supabase project url with any of the endpoints doesn't work.

Thanks!

**Comment by soedirgo** [comment]
Hey @andrewcoelho, it should be `https://<...>.supabase.co/auth/v1/`.

**Comment by zernonia** [comment, marked as answer]
Here's an example on how to use gotrue endpoint: https://supabase.io/docs/learn/auth-deep-dive/auth-gotrue, which is `'https://<project-ref>.supabase.co/auth/v1/`
